index.html

```diff
@@ -158,6 +158,9 @@ <h2>Scope</h2>
 
         <p class="issue">The community group has identified, in its <a href="https://www.w3.org/2021/12/rdf-star.html">final report</a>, which recommendations should be updated, and a possible path for updating them. The Working Group may however reconsider this and proceed differently from the Community Group. For every recommendation updated by this Working Group, the pending editorial errata will also be addressed. The Working Group will also consider <a href="https://www.w3.org/2020/Process-20200915/#allow-new-features">allowing new features</a> in these recommendations, according to <a href="https://www.w3.org/2020/Process-20200915/#revised-rec-features">Section 6.2.11.4 of the W3C process</a>, in order to render future evolutions easier.</p>
 
+        <p class="issue">The group will ensure that any RDF 1.1 data remains valid in this new version. Furthermore, any RDF or RDFS entailment drawn under RDF 1.1 semantics should also remain valid in this new version.</p>
```

Comment:
I don't think we need the "Furthermore" text.

if you want some additional text, it should be general for all entailment regimes based on the principles of RDF (e.g. monotonicity), rather than name RDFS but I don't see any changes that would have an effect if the "data remains valid".

"will ensure" => "should ensure" if people want a specific weight to the statement. 

"should" because of errata and, based on RDF 1.1, things learnt by revisiting specs makes "must" extremely strong. "clarification" and "change" can a matter of point of view.

Reply:
> I don't think we need the "Furthermore" text.

I actually picked that up from the RDF 1.1 charter. It seemed to make sense...